Example/Example/experience.json

```diff
@@ -0,0 +1,355 @@
+                    "width": 936,
+                    "url": "https://images-rover-io.imgix.net/uploads/c4ca4238a0b923820dcc509a6f75849b/5156d40b-11ed-40c2-8f41-9ed681842c61-tex-mex-lasagna-main.png"
+                  }
+                }
+              ],
+              "questionStyle": {
+                "color": {
+                  "red": 129,
+                  "green": 129,
+                  "blue": 129,
+                  "alpha": 1
+                },
+                "font": {
+                  "size": 16,
+                  "weight": "LIGHT"
+                },
+                "textAlignment": "CENTER"
+              },
+              "optionStyle": {
+                "color": {
+                  "red": 10,
+                  "green": 10,
+                  "blue": 10,
+                  "alpha": 1
+                },
+                "opacity": 1,
+                "border": {
+                  "color": {
+                    "red": 129,
+                    "green": 129,
+                    "blue": 129,
+                    "alpha": 1
+                  },
+                  "radius": 20,
+                  "width": 0
+                },
+                "background": {
+                    "color": {
+                        "red": 176,
+                        "green": 255,
+                        "blue": 255,
+                        "alpha": 1
+                    },
+                    "contentMode": "ORIGINAL",
+                    "scale": "X1"
+                },
+                "font": {
+                  "size": 16,
+                  "weight": "LIGHT"
+                },
+                "textAlignment": "LEFT",
+                "resultFillColor": {
+                  "red": 129,
+                  "green": 129,
+                  "blue": 129,
+                  "alpha": 1
+                },
+                "verticalSpacing": 16,
+                "horizontalSpacing": 16
+              }
+            }
+          ],
+          "height": {
+            "__typename": "HeightIntrinsic"
+          },
+          "id": "HJqhJnlc",
+          "name": "Row",
+          "keys": {},
+          "tags": []
+        }
+      ],
+      "statusBar": {
+        "style": "LIGHT",
+        "color": {
+          "red": 245,
+          "green": 15,
+          "blue": 15,
+          "alpha": 1
+        }
+      },
+      "titleBar": {
+        "backgroundColor": {
+          "red": 247,
+          "green": 64,
+          "blue": 64,
+          "alpha": 1
+        },
+        "buttons": "CLOSE",
+        "buttonColor": {
+          "red": 241,
+          "green": 189,
+          "blue": 189,
+          "alpha": 1
+        },
+        "text": "What's Cooking?",
+        "textColor": {
+          "red": 255,
+          "green": 255,
+          "blue": 255,
+          "alpha": 1
+        },
+        "useDefaultStyle": false
+      },
+      "keys": {},
+      "tags": []
+    }
+  ],
+  "keys": {},
+  "tags": []
+}
```

Comment:
this will also be removed before release. just makes working on the test bench easier.